Directory.Packages.props

```diff
@@ -144,24 +144,24 @@
     <PackageVersion Include="Microsoft.DotNet.Build.Tasks.Installers" Version="8.0.0-beta.23564.4" />
     <PackageVersion Include="System.CommandLine" Version="2.0.0-beta4.24324.3" />
     <!-- unit test dependencies -->
-    <PackageVersion Include="bUnit" Version="1.31.3" />
-    <PackageVersion Include="JsonSchema.Net" Version="7.2.2" />
+    <PackageVersion Include="bUnit" Version="1.33.3" />
+    <PackageVersion Include="JsonSchema.Net" Version="7.2.3" />
     <PackageVersion Include="Microsoft.DotNet.RemoteExecutor" Version="$(MicrosoftDotNetRemoteExecutorPackageVersion)" />
     <PackageVersion Include="Microsoft.DotNet.XUnitExtensions" Version="$(MicrosoftDotNetXUnitExtensionsPackageVersion)" />
     <PackageVersion Include="Microsoft.Extensions.Diagnostics.Testing" Version="$(MicrosoftExtensionsDiagnosticsTestingPackageVersion)" />
     <PackageVersion Include="Microsoft.NET.Runtime.WorkloadTesting.Internal" Version="$(MicrosoftNETRuntimeWorkloadTestingInternalVersion)" />
-    <PackageVersion Include="Microsoft.Playwright" Version="1.46.0" />
-    <PackageVersion Include="Swashbuckle.AspNetCore" Version="6.7.3" />
+    <PackageVersion Include="Microsoft.Playwright" Version="1.47.0" />
+    <PackageVersion Include="Swashbuckle.AspNetCore" Version="6.9.0" />
     <PackageVersion Include="Testcontainers.MongoDb" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.MsSql" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.MySql" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.PostgreSQL" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.RabbitMq" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.Redis" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.Nats" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.Milvus" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.Oracle" Version="$(TestcontainersPackageVersion)" />
-    <PackageVersion Include="Testcontainers.Elasticsearch" Version="$(TestcontainersPackageVersion)" />
+    <PackageVersion Include="Testcontainers.MsSql" Version="3.10.0" />
+    <PackageVersion Include="Testcontainers.MySql" Version="3.10.0" />
+    <PackageVersion Include="Testcontainers.PostgreSQL" Version="3.10.0" />
+    <PackageVersion Include="Testcontainers.RabbitMq" Version="3.10.0" />
+    <PackageVersion Include="Testcontainers.Redis" Version="3.10.0" />
+    <PackageVersion Include="Testcontainers.Nats" Version="3.10.0" />
+    <PackageVersion Include="Testcontainers.Milvus" Version="3.10.0" />
+    <PackageVersion Include="Testcontainers.Oracle" Version="3.10.0" />
+    <PackageVersion Include="Testcontainers.Elasticsearch" Version="3.10.0" />
```

Comment:
Revert this and update the property above.

Reply:
If you see there is one testcontainer that didn't actually change which is why I didn't, but I could of course do the other way and update the property and just hardcode the one that didn't update